ssh/tailssh/testcontainers/Dockerfile

```diff
@@ -1,19 +1,22 @@
 ARG BASE
 FROM ${BASE}
 
+ARG BASE
+
 RUN echo "Install openssh, needed for scp."
-RUN apt-get update -y && apt-get install -y openssh-client
+RUN if echo "$BASE" | grep "ubuntu:"; then apt-get update -y && apt-get install -y openssh-client; fi
```

Comment:
Possibly there's a cleaner way to distinguish between ubuntu and alpine. I had considered making two separate Dockerfiles, but I didn't want to have to keep the test cases synced between them.